### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by brucellino**
The code of conduct is especially important for this repo. This service is _all about_ community, so we need to have a proper discussion and look over this carefully.

I'm adding the generic code of conduct here, as a starting point.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (gwarf) [inline CODE_OF_CONDUCT.md]: So we are allowed to harass people on oser topics?

Comment (enolfc) [reply CODE_OF_CONDUCT.md]: No, but I understand those are the typical topics and is good that people are reminded.

**Comment by enolfc** [comment]
LGTM

Sure there are laws, but laws don't remove nasty behaviour just by existing, so I think is nice we make an explicit call for being nice. 

@gwarf I'm aware this political correctness stuff sometimes feels like avoiding free speech, but this is a EGI discussion forum, you can discuss anything you want about EGI, just be nice to other people.

**Comment by brucellino** [comment]
This issue was moved to https://community.egi.eu/t/create-code-of-conduct-md/79

**Comment by brucellino** [comment]
I've moved this to discourse. Thank you very much for the input, I'll send a new PR with this stuff taken into account, and we can discuss this over at https://community.egi.eu/t/create-code-of-conduct-md/79

**Pull request closed without merging**